app/templates/orders/order_block.hbs

```diff
@@ -1,31 +1,82 @@
-<li>
+<li class="order_block">
 
-    <div class="row state_details">
-      <div class="small-5 columns">
-        {{order.code}} - {{order.status}}
-      </div>
-      <div class="small-4 columns">
-        {{order.designatedOrdersPackages.length}} <i class="fa fa-shopping-basket"></i>
-        {{order.dispatchedOrdersPackages.length}} <i class="fa fa-ship"></i>
+    <div class="row order_header">
+
+      <div class="small-4 columns order_transport"> 
+        <i class="fa fa-lg fa-{{ order.transportIcon }}"></i> {{ order.transportLabel }}
```

Comment:
I think we can remove spaces from curly braces

Reply:
Just for readability